### PR TITLE
[Vue] Rename AngularJSTests suite to ClientTests.

### PR DESCRIPTION
**Pull request by diosmosis**
As title.

Merge w/ https://github.com/matomo-org/matomo/pull/19502

### CODE REVIEW

**Comment by sgiehl** [comment]
@diosmosis merging this one will break tests on 4.x-dev, right? Maybe we should keep the angularjs for bc here?

**Comment by diosmosis** [comment]
we could check for both here

File: setup_mysql_tmpfs.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [ "$SKIP_PIWIK_TEST_PREPARE" == "1" ] || [ "$TEST_SUITE" == "AngularJSTests" ] || [ "$TEST_SUITE" == "UnitTests" ] || [ "$TEST_SUITE" == "JavascriptTests" ]; then
+if [ "$SKIP_PIWIK_TEST_PREPARE" == "1" ] || [ "$TEST_SUITE" == "ClientTests" ] || [ "$TEST_SUITE" == "UnitTests" ] || [ "$TEST_SUITE" == "JavascriptTests" ]; then
```
Comment (sgiehl) [inline setup_mysql_tmpfs.sh:3]:
```suggestion
if [ "$SKIP_PIWIK_TEST_PREPARE" == "1" ] || [ "$TEST_SUITE" == "AngularJSTests" ] || [ "$TEST_SUITE" == "ClientTests" ] || [ "$TEST_SUITE" == "UnitTests" ] || [ "$TEST_SUITE" == "JavascriptTests" ]; then
```

File: setup_webserver.sh [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-if [ "$SKIP_PIWIK_TEST_PREPARE" == "1" ] || [ "$TEST_SUITE" == "AngularJSTests" ] || [ "$TEST_SUITE" == "UnitTests" ]; then
+if [ "$SKIP_PIWIK_TEST_PREPARE" == "1" ] || [ "$TEST_SUITE" == "ClientTests" ] || [ "$TEST_SUITE" == "UnitTests" ]; then
```
Comment (sgiehl) [inline setup_webserver.sh:2]:
```suggestion
if [ "$SKIP_PIWIK_TEST_PREPARE" == "1" ] || [ "$TEST_SUITE" == "AngularJSTests" ] || [ "$TEST_SUITE" == "ClientTests" ] || [ "$TEST_SUITE" == "UnitTests" ]; then
```

**Review by sgiehl — CHANGES_REQUESTED** [review]
Those should remain for BC reasons

**Pull request merged**